content/en/docs/Concepts/operators-on-cluster.md

```diff
@@ -0,0 +1,21 @@
+# When to use operators on a cluster
+
+
+Operators are good for automating the knowledge of how to operate a complex system. Operators are a high-privilege component and by design, they run persistently inside your cluster. An operators can handle features like automatic scaling in response to load, backup and restore, integration with metric systems like Prometheus, and even failure detection and automatic tuning in response to usage patterns.
+
+# How does OLM help you use operators on a cluster?
```

Comment:
I think you should revisit this question because the rest of this document does not answer this question in my eyes. I don't see how OLM helps me use operators on a cluster.